packages/components/organisms/f-checkout/src/tenants/en-GB.js

```diff
@@ -51,51 +51,51 @@ const messages = {
 
         checkoutIssues: {
             MINIMUM_ORDER_VALUE_NOT_MET: {
-                title: 'Your order does not meet the minimum spend required',
-                message: 'The restaurant might have run out or updated its menu while you were ordering, sorry. You can go back to check your order and choose another item.',
-                buttonText: 'Back to Order'
+                title: 'Minimum spend not reached',
+                message: 'We\'re sorry, but the minimum spend for your order has not been reached. Please amend your order.',
+                buttonText: 'Back to order'
             },
 
             RESTAURANT_NOT_TAKING_ORDERS: {
-                title: 'We couldn\'t send your order',
-                message: 'The restaurant went offline, so we couldn\'t send your order to them, sorry. Hopefully it\'s only temporary, please try again.',
-                buttonText: 'Retry'
+                title: 'Restaurant not taking orders',
+                message: 'We\'re sorry, but the restaurant is not currently taking orders. Please choose another restaurant to order from.',
```

Comment:
You can use `‘` here rather than the standard `dumb` apostrophe. Saves escaping it and it's also better topographically when it gets displayed on screen.

See these links if you're interested:
- https://www.presentationzen.com/presentationzen/2009/08/apostrophes-and-quotation-marks.html#:~:text=To%20get%20a%20proper%20apostrophe,by%20the%20best%20of%20them.
- https://www.fastcompany.com/3018353/be-smart-dont-use-dumb-quote-marks

Reply:
Yep, I remember you saying. Will update now. Thanks